src/wp-includes/media-template.php

```diff
@@ -1542,21 +1542,31 @@ function wp_print_media_templates() {
 	</script>
 
 	<?php // Template for the Site Icon preview, used for example in the Customizer. ?>
-	<script type="text/html" id="tmpl-site-icon-preview">
-		<h2><?php _e( 'Preview' ); ?></h2>
-		<strong aria-hidden="true"><?php _e( 'As a browser icon' ); ?></strong>
-		<div class="favicon-preview">
-			<img src="<?php echo esc_url( admin_url( 'images/' . ( is_rtl() ? 'browser-rtl.png' : 'browser.png' ) ) ); ?>" class="browser-preview" width="182" height="" alt="" />
-
-			<div class="favicon">
-				<img id="preview-favicon" src="{{ data.url }}" alt="<?php esc_attr_e( 'Preview as a browser icon' ); ?>" />
+	<script type="text/html" id="tmpl-site-icon-preview-crop">
+		<style>
+			:root{
+				--site-icon-url: url( "{{ data.url }}" );
+			}
+		</style>
+		<h2><?php _ex( 'Site Icon Preview', 'noun' ); ?></h2>
+		<p aria-hidden="true"><?php _e( 'As a browser icon and an app icon.' ); ?></p>
```

Comment:
I wouldn't hide this paragraph from assistive technologies. Although the two following images do have meaningful alt text, letting this text be read by screen readers doesn't harm.

Reply:
Removed aria-hidden.